src/hound/cmds/houndd/main.go

```diff
@@ -190,14 +191,13 @@ func makeSearchers(
 			s, err = searcher.New(path, repo)
 		}
 
-		if err != nil {
-			return nil, err
+		if err == nil {
+			m[strings.ToLower(name)] = s
 		}
-		m[strings.ToLower(name)] = s
 
 	}
 
-	return m, nil
+	return m, err
```

Comment:
Instead of returning the last error, you should probably return a []error or maybe even a map[string]error that maps Repo to error.